docs/library/neopixel.rst

```diff
@@ -54,17 +54,17 @@ Pixel access methods
     Sets the value of all pixels to the specified *pixel* value (i.e. an
     RGB/RGBW tuple).
 
-.. method:: NeoPixel.__len__()
+.. method:: len(np)
```

Comment:
There is no method `Neopixel.len`

Reply:
I realise it's not a method but the `.. method::` directive is as much about formatting as anything else.  What would you suggest.  As I say I was using pythons documentation as a guide.

Reply:
I recommend to leave the docs unchanged - as changing them will likely break the hyperlinks in the documentation , and surely break the generation of the micropython-stubs, which actually parse the directives. 
Please see [my comment](https://github.com/micropython/micropython/pull/16764#issuecomment-2661558578) for recommendations